tests/sqllogictests/suites/ee/04_ee_inverted_index/04_0000_inverted_index_base.test

```diff
@@ -308,175 +308,176 @@ SELECT id, score(), title FROM books WHERE query('title:pyth', 'fuzziness=2') OR
+SELECT id, score(), title FROM books WHERE query('title:+设计 -实现') ORDER BY id
 ----
-10 2.0477252 前端架构设计
-8 1.8123543 Rust程序设计（第2版）
+8 1.0 Rust程序设计（第2版）
+10 1.0 前端架构设计
 
 query IFT
-SELECT id, score(), title FROM books WHERE query('title:+设计 实现') ORDER BY score() DESC
+SELECT id, score(), title FROM books WHERE query('title:+设计 实现') ORDER BY id
 ----
-9 5.063791 Vue.js设计与实现
-10 2.0477252 前端架构设计
-8 1.8123543 Rust程序设计（第2版）
+8 1.0 Rust程序设计（第2版）
+9 1.0 Vue.js设计与实现
+10 1.0 前端架构设计
 
 query IFT
-SELECT id, score(), title FROM books WHERE query('title:python^5 description:chatgpt^2.1') ORDER BY score() DESC
+SELECT id, score(), title FROM books WHERE query('title:python^5 description:chatgpt^2.1') ORDER BY id
 ----
-13 7.890149 Learn AI-Assisted Python Programming: With GitHub Copilot and ChatGPT
-2 7.1890326 Python深度学习（第2版）
-14 5.509352 Building Recommendation Systems in Python and JAX
-11 5.263399 OpenAI GPT For Python Developers, 2nd Edition
-6 4.8319726 Flask Web开发：基于Python的Web应用开发实战（第2版）
-1 4.732555 这就是ChatGPT
-12 4.325484 Developing Apps with GPT-4 and ChatGPT
-3 3.106897 大模型应用开发极简入门
+1 1.0 这就是ChatGPT
+2 1.0 Python深度学习（第2版）
+3 1.0 大模型应用开发极简入门
+6 1.0 Flask Web开发：基于Python的Web应用开发实战（第2版）
+11 1.0 OpenAI GPT For Python Developers, 2nd Edition
+12 1.0 Developing Apps with GPT-4 and ChatGPT
+13 1.0 Learn AI-Assisted Python Programming: With GitHub Copilot and ChatGPT
+14 1.0 Building Recommendation Systems in Python and JAX
 
 query IFT
-SELECT id, score(), title FROM books WHERE query('title:(设计 实现)^5 description:(学习 +神经网络)^1.1') ORDER BY score() DESC
+SELECT id, score(), title FROM books WHERE query('title:(设计 实现)^5 description:(学习 +神经网络)^1.1') ORDER BY id
 ----
-9 25.318954 Vue.js设计与实现
-4 22.395063 白话深度学习的数学
-10 10.238626 前端架构设计
-8 9.061771 Rust程序设计（第2版）
-
+2 1.0 Python深度学习（第2版）
+4 1.0 白话深度学习的数学
+7 1.0 Apache Pulsar实战
+8 1.0 Rust程序设计（第2版）
+9 1.0 Vue.js设计与实现
+10 1.0 前端架构设计
 
 # index without optional filters and index rocord is basic
-statement ok
-CREATE OR REPLACE INVERTED INDEX idx2 ON books(title, author, description) tokenizer = 'chinese' index_record='basic'
-
-statement ok
-REFRESH INVERTED INDEX idx2 ON books
-
-query IFT
-SELECT id, score(), title FROM books WHERE match('title^5, description^1.2', 'python') ORDER BY score() DESC
-----
-2 8.192706 Python深度学习（第2版）
-6 6.235875 Flask Web开发：基于Python的Web应用开发实战（第2版）
-14 5.4896193 Building Recommendation Systems in Python and JAX
-11 5.2801366 OpenAI GPT For Python Developers, 2nd Edition
-13 4.2964296 Learn AI-Assisted Python Programming: With GitHub Copilot and ChatGPT
-4 1.5421177 白话深度学习的数学
-3 1.3799851 大模型应用开发极简入门
-12 1.3110648 Developing Apps with GPT-4 and ChatGPT
-7 1.2791233 Apache Pulsar实战
-
-query IFT
-SELECT id, score(), title FROM books WHERE query('title:设计 AND 实现 OR 实战') ORDER BY score() DESC
-----
-9 5.027091 Vue.js设计与实现
-7 2.2837715 Apache Pulsar实战
-5 1.7452873 BERT基础教程：Transformer大模型实战
-6 1.2672173 Flask Web开发：基于Python的Web应用开发实战（第2版）
+#statement ok
```

Comment:
These tests no used any more?

Reply:
Temporarily modified, the following PR implementation of the relevant functions will be restored